tests/Doctrine/Tests/DBAL/Functional/DataAccessTest.php

```diff
@@ -553,8 +551,6 @@ public function testDateArithmetics()
         $row = $this->_conn->fetchAssoc($sql);
         $row = array_change_key_case($row, CASE_LOWER);
 
-        $diff = (strtotime('2010-01-01') - strtotime(date('Y-m-d'))) / 3600 / 24;
-        self::assertEquals($diff, $row['diff'], "Date difference should be approx. ".$diff." days.", 1);
```

Comment:
This "approx." and `$round` hides the existing issue. The assertion is reworked in `DateExpressionTest`.